### PR TITLE
Css basics

### PR DESCRIPTION
**Pull request by Scot-MacDonald**

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
Nice job, well done!

File: css-basics_pokemon-page/css/styles.css [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+/* Task 2: Let's work on some basic stylings that we want to apply to all elements of your code via inheritance. Which selector do we need for this? */
+
+/* add selector */
+* {
```
Comment (ghost) [inline css-basics_pokemon-page/css/styles.css]: I'm not done with it yet but I wouldn't use the selector * twice in the code. Maybe body?

File: css-basics_pokemon-page/css/styles.css [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+/* ------- Headings Part 1 ------- */
+
+/* Task 3: To set the line height to 1.5 is good for texts, because of accessibility concerns, but for the headings it seems too much. Let's change this by giving our headings a line height of 1.2: */
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
```
Comment (ghost) [inline css-basics_pokemon-page/css/styles.css]: If I'm not mistaken there's only the h1 and h2 selectors. :)

File: css-basics_pokemon-page/css/styles.css (hunk 1 above)
Comment (Scot-MacDonald) [reply css-basics_pokemon-page/css/styles.css]: Thanks Victor!!

File: css-basics_pokemon-page/css/styles.css (hunk 2 above)
Comment (Scot-MacDonald) [reply css-basics_pokemon-page/css/styles.css]: done

**Pull request closed without merging**